adapters/beachfront/beachfront.go

```diff
@@ -103,11 +100,15 @@ type beachfrontResponseSlot struct {
 	Adm   string  `json:"adm"`
 }
 
-type beachfrontVideoBidExtension struct {
+type videoBidExtension struct {
 	Duration int `json:"duration"`
 }
 
+var one int8 = 1
+var zero int8 = 0
+
 func (a *BeachfrontAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var reqs = make([]*adapters.RequestData, 0)
```

Comment:
We should size this to the max expected size, so we don't have to keep allocating larger and larger slices on the appends. Something like:
```
var reqCount = len(beachfrontRequests.ADMVideo) + len(beachfrontRequests.NurlVideo) + 1
var reqs = make([]*adapters.RequestData, 0, reqCount)
```